pandas/tseries/tools.py

```diff
@@ -349,7 +350,11 @@ def _convert_listlike(arg, box, format):
     elif com.is_list_like(arg):
         return _convert_listlike(arg, box, format)
 
-    return _convert_listlike(np.array([ arg ]), box, format)[0]
+    try:
+        return _convert_listlike(np.array([ arg ]), box, format)[0]
+    except ValueError as e:
+        if not coerce:
```

Comment:
you are catching an incorrect exception (something else is wrong) - it's not an expected exception